data_buffer.cpp

```diff
@@ -355,7 +355,7 @@ real_t DataBuffer::add_positive_unit_real(real_t p_input, CompressionLevel p_com
 #endif
 	ERR_FAIL_COND_V(is_reading == true, p_input);
 
-	const int bits = get_bit_taken(DATA_TYPE_UNIT_REAL, p_compression_level);
+	const int bits = get_bit_taken(DATA_TYPE_POSITIVE_UNIT_REAL, p_compression_level);
```

Comment:
what's the difference here between `DATA_TYPE_UNIT_REAL` and `DATA_TYPE_POSITIVE_UNIT_REAL`? 
Does it influences in any way our ability to use negative numbers?

Reply:
When you use `add_positive_unit_real` you can't submit negative numbers. In the case you need to submit negative and positive numbers, you have to use `add_unit_real`.